fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_coffin.dm

```diff
@@ -40,17 +40,17 @@
 
 /obj/structure/closet/crate/coffin/securecoffin
 	name = "secure coffin"
-	desc = "For those too scared of having their place of rest disturbed."
+	desc = "For insecure suckers to safely slumber in their secure sleeping space, surprisingly, this shelter's spacious sepulchre offers somewhat superior security"
```

Comment:
```suggestion
	desc = "Provides a secure experience against most hazards, for when you can't handle the stakes."
```

Reply:
i think this is funnier because stakes is also something in the game that bloodsuckers dislike